### DISCUSSION TITLE
HIT OS: Operating System - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#470] [Announcements]
# HIT OS: Operating System - CS自学指南

CS自学指南

https://csdiy.wiki/%E6%93%8D%E4%BD%9C%E7%B3%BB%E7%BB%9F/HITOS/

**Comment by SiyuanYue** [comment]
这是2020 后本部 刘国军老师的课程主页：http://os.guojunhit.cn/            实验基于 linux 0.11/0.12 代码
这是哈深那边的课程主页： http://hitsz-cslab.gitee.io/os-labs/             实验基于xv6

**Reply by JP-ZHU** [reply]
课程主页里面好像只列出了课件的具体条目没给出课件的内容，请问是跟李治军老师的课件内容差不多嘛？

**Comment by AaronWangggg** [comment]
很棒的工作 爱来自瓷器❤

**Comment by keira10000** [comment]
谢谢

**Comment by chuilishi** [comment]
太棒了

**Comment by sarendis56** [comment]
看这门课之前最好先了解汇编语言

**Reply by Uesugi2203** [reply]
有无推荐的汇编快速入门课程或书籍?

**Comment by Larry-Cui** [comment]
别的还没学，先把这个网站的模板material搞到了，也算一大收获😅

**Comment by Liangduoyuan** [comment]
内容非常不错，谢谢博主

**Comment by fkgkdfgy** [comment]
根据推荐刷完了CS61A，看待Python 感觉更加深入了

**Reply by fkgkdfgy** [reply]
根据推荐刷完了nand2tetris，对于非科班的人来说，像是拆开了黑盒子。

**Reply by lijun135** [reply]
你是怎么进入CS61A的课程里的呀？我注册后还是没有办法看到课程

**Reply by fkgkdfgy** [reply]
可以直接google CS61A fa20 就可以

**Reply by AlanXiangYuWu** [reply]
太棒了多谢楼主，一个问题，我原本搜索关键词是CS61A，出的那个需要Berkeley account，哪个是最新版是吧，老版能直接看

**Comment by bencreater** [comment]
太感谢了

**Comment by Abenoroth** [comment]
非常感谢

**Comment by Lezy233** [comment]
魔兽世界 非常好工作 爱来自瓷器 ❤️

**Comment by gaomingruc** [comment]
谢谢你。我准备用你这个网站自学下cs。我之前学过python，平时用pandas、matplotlib、seaborn、statsmodels完成点上班的工作。之前想深入学学其他有意思的内容，比如爬虫、搭网站、AI、树莓派，但因为缺乏使用场景，都没有学进去。看你写的这么好，我再试试吧～

**Comment by scoteweid** [comment]
大佬，我希望我们搞技术的不要受到政治的价值影响，不要考虑什么崇洋媚外与否，我们应该尊重事实，尊重理性，事实就是在CS这块国外好，国内垃圾，我觉得没什么好避讳的。

**Reply by valorantJoln** [reply]
作者的内容是很多人看的，所以他考虑的是适合不同观点立场的人对国内外的资源理解，万一有个别极端人又来以此说点啥处理就很麻烦，早点狗头早点解决😉

**Reply by ElysiaIWTB** [reply]
你只需要打着所谓技术崇拜的名义站在道德的制高点上指指点点就够了, 而别人要考虑的就多了. 毕竟事不关己高高挂起不是么?

**Comment by Danie1Yang** [comment]
想问一下大佬们遇到没有字幕的课程是怎么解决的？还是大佬们的听力水平足够应付课程了

**Reply by Wh1tew0lf-sea** [reply]
推荐一个好用的翻译 Language Reactor，适用于大多数的YouTube什么的视频网站，我用这个学习完了MIT 6.s081

**Comment by syccxdr** [comment]
thanks for your hard work.

**Comment by AlanXiangYuWu** [comment]
是的，Chrome插件库里面会有类似的插件Sent from my iPhoneOn Aug 1, 2023, at 20:51, Wh1tew0lf ***@***.***> wrote:﻿
推荐一个好用的翻译 Language Reactor，适用于大多数的YouTube什么的视频网站，我用这个学习完了MIT 6.s081

**Comment by silenter-su** [comment]
看起来不错的网站哦。

**Comment by hadonga** [comment]
当年从硕士才开始学cs，又遇到教授放养，摸爬滚打读到博士毕业。但是到现在很多部分都不成体系，如果按照这个教程逐个攻破的话，肯定有质的提高。感谢作者的整理，请作者喝几杯星巴克~

**Comment by wabi-liu** [comment]
在跨考的今年碰到了这个网站，我想如果能早两年遇到我会有方向感一些。很感激作者，太详细了。我今年没太多时间来学了，等读研究生的时候我一定要过一遍。

**Comment by JamieOw119** [comment]
我的天哪，太扎实了

**Comment by Alex200411** [comment]
这篇文章怎么加入obsidian啊

**Comment by 1135908055** [comment]
给大家推荐个简单的科学上网方式https://user.vipthree.xyz/register?aff=83859

**Comment by yjxzx** [comment]
我为什么总是收到你的消息？
________________________________
发件人: 1135908055 ***@***.***>
发送时间: 2023年8月9日 0:48
收件人: PKUFlyingPig/cs-self-learning ***@***.***>
抄送: yjxzx ***@***.***>; Manual ***@***.***>
主题: Re: [PKUFlyingPig/cs-self-learning] HIT OS: Operating System - CS自学指南 (Discussion #470)


给大家推荐个简单的科学上网方式https://user.vipthree.xyz/register?aff=83859

―

**Reply by sarendis56** [reply]
在本repository主页中选择右上角watch，点击ignore

**Comment by silenter-su** [comment]
github中取消消息通知？


85358830
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "PKUFlyingPig/cs-self-learning"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年8月9日(星期三) 下午2:30
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [PKUFlyingPig/cs-self-learning] HIT OS: Operating System - CS自学指南 (Discussion #470)





 我为什么总是收到你的消息？ 
 ________________________________ 
 发件人: 1135908055 ***@***.***&gt; 
 发送时间: 2023年8月9日 0:48 
 收件人: PKUFlyingPig/cs-self-learning ***@***.***&gt; 
 抄送: yjxzx ***@***.***&gt;; Manual ***@***.***&gt; 
 主题: Re: [PKUFlyingPig/cs-self-learning] HIT OS: Operating System - CS自学指南 (Discussion #470) 
 
 
 给大家推荐个简单的科学上网方式https://user.vipthree.xyz/register?aff=83859 
 
 ―

**Comment by yjxzx** [comment]
是关于cs的交谈。我试试。
________________________________
发件人: Hyle33ies ***@***.***>
发送时间: 2023年8月9日 14:33
收件人: PKUFlyingPig/cs-self-learning ***@***.***>
抄送: yjxzx ***@***.***>; Manual ***@***.***>
主题: Re: [PKUFlyingPig/cs-self-learning] HIT OS: Operating System - CS自学指南 (Discussion #470)


在本repository主页中选择右上角watch，点击ignore

―

**Comment by rookiewsj** [comment]
大佬们，有没有推荐的汇编语言的课程

**Reply by DuoHong123** [reply]
王爽《汇编语言》

**Reply by nickzh2024** [reply]
感觉王爽那个内容不够，搞OS需要保护模式的知识，王爽那个只讲了实模式

**Comment by mowang111** [comment]
请问有没有哈工大这个实验的交流群啊，有点问题想请教一下

**Comment by xylytx** [comment]
看完了b站22的，第一章讲代码开始就挺晦涩的了，一些示意图也没有网上的图或者动画好理解，没有做lab，个人感觉推荐指数没有那么高，风格和其他学校教的差不多比较按部就班，推荐《穿越操作系统迷雾》，这个书对于零基础的要好理解很多
